### ISSUE TITLE
"correlate"/"correlation" confusion in doe_lhs.py

### ISSUE THREAD

**Issue by Arnoques** [tisimst/pyDOE#4]
Just a heads-up:

There's a mixup with the name for the correlation criterion for the Latin Hypercube. In `doe_lhs.py`, the documentation says `"correlation"` but the example says `'correlate'`. The check in the code uses `'correlation'`, but later the if that actually selects the used function uses `'correlate'`.

```
36:        "centermaximin" or "cm", and "correlation" or "corr". If no value 
88:        >>> lhs(4, samples=5, criterion='correlate', iterations=10)
98:            'centermaximin', 'cm', 'correlation', 
116:        elif criterion.lower() in ('correlate', 'corr'):
```

Thanks!

**Comment by tisimst** [comment]
You are correct. They all should be "correlate". Thank you for bringing that to my attention!

**Comment by cjekel** [comment]
I'd be happy to send a PR for this. I just ran into this issue today.

**Comment by saudzahirr** [comment]
Thanks for bringing this up. The issue has now been resolved. Thank you for using pyDOE and for contributing to the project!
